test/kb_trimmomatic_server_test.py

```diff
@@ -99,6 +99,8 @@ def upload_file_to_shock(cls, file_path):
 
         if file_path is None:
             raise Exception("No file given for upload to SHOCK!")
+        if not file_path.startswith('/'):
+            file_path = os.path.join('/kb/module/test', file_path)
```

Comment:
if you're going to join a path, should probably join all the members: (os.sep, "kb", "module", ....)